hack/csv-generator.go

```diff
@@ -83,7 +83,6 @@ func init() {
 	rootCmd.Flags().BoolVar(&f.dumpCRDs, "dump-crds", false, "Dump crds to stdout")
 
 	rootCmd.MarkFlagRequired("csv-version")
-	rootCmd.MarkFlagRequired("namespace")
```

Comment:
I would leave this field as required, and pass the `kubevirt` namespace to the csv-generator

Reply:
That namespace field doesn't seem to be used anywhere, the value passed to it doesn't end up in the final yaml, and csv-generator runs happily without it.
Maybe it's there for future use? In any case, it doesn't feel like it should be "required" just yet.